### PR TITLE
Setup py

### PR DESCRIPTION
**Pull request by felipaoo**
agregé setup.py #6 
y se ignora .idea/

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+with open('README.md', 'r') as f:
+    long_description = f.read()
+
+
+setuptools.setup(
+    name='arcus-python',
+    version='0.0.1',
+    author='Cuenca',
+    author_email='dev@cuenca.com',
+    description='Regalii API Client',
```
[the remark this answers is not in the document]
Comment (felipaoo) [reply setup.py]: ok, so Arcus API Client is ok?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            'pytest>=3',
+            'pycodestyle'
+        ]
+    },
+    classifiers=[
+        'Programming Language :: Python :: 3',
+        'License :: OSI Approved :: MIT License',
+        'Operating System :: OS Independent',
+    ]
+)
```
[the remark this answers is not in the document]
Comment (felipaoo) [reply setup.py]: ok, I'll fix it

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import setuptools
+
+
+with open('README.md', 'r') as f:
+    long_description = f.read()
+
+
+setuptools.setup(
+    name='arcus-python',
```
Comment (matin) [inline setup.py]: this should be `arcus` to be consistent with https://pypi.org/project/arcus

Comment (felipaoo) [reply setup.py]: done

**Pull request merged**